airbyte-webapp/src/components/connectorBuilder/Builder/Builder.module.scss

```diff
@@ -18,5 +18,6 @@
 .form {
   flex: 1;
   padding: variables.$spacing-xl;
+  padding-right: 80px;
```

Comment:
nit: apply this padding to the left as well, so that the form is centered in its containing div

Reply:
80px is a lot of horizontal screen space to lose, instead I made this styling conditional based on whether there are multiple streams in the first place. Also solves the issue of not having an unnecessary gap on user input and global views.

Reply:
Nice, I like that approach 👍